modules/bazel_compile_commands/0.5.0/MODULE.bazel

```diff
@@ -0,0 +1,29 @@
+module(
+    name = "bazel_compile_commands",
```

Comment:
This name is kind of general. Do you mind prefixing it with your github user name? For example, `wolfd_bazel_compile_commands`?